services/stiprelayer/db/sql/base/store.go

```diff
@@ -20,6 +22,31 @@ func (s *Store) GetSTIPTransactionsNotRebated(ctx context.Context) ([]*db.STIPTr
 	return stipTransactions, nil
 }
 
+// GetTotalArbRebated gets the total amount of arb rebated for a given address.
+func (s *Store) GetTotalArbRebated(ctx context.Context, address string) (*big.Int, error) {
+	var stipTransactions []*db.STIPTransactions
+
+	// Fetch all transactions that have been rebated for the given address
+	result := s.db.WithContext(ctx).
+		Where("rebated = ?", true).
+		Where("address = ?", address).
+		Find(&stipTransactions)
+	if result.Error != nil {
+		return nil, result.Error
+	}
+
+	// Compute the sum of arb rebated across all transactions
+	totalRebated := big.NewInt(0)
+	for _, stipTransaction := range stipTransactions {
+		rebatedAmount, ok := new(big.Int).SetString(stipTransaction.ArbAmountRebated, 10)
```

Comment:
I don't think this converts from 18 decimals accurately, maybe:

```
weiInOneEther := big.NewInt(0).Exp(big.NewInt(10), big.NewInt(18), nil)
rebatedAmountWei, _ := new(big.Int).SetString(stipTransaction.ArbAmountRebated, 10)
rebatedAmountEther := new(big.Float).Quo(new(big.Float).SetInt(rebatedAmountWei), new(big.Float).SetInt(weiInOneEther))
```

Reply:
I thought the `arb_amount_rebated` field was already in wei units, hence the `String` type? Since we are dealing with wei units in relayer I dont think this conversion is necessary?

Reply:
yep you're right lgtm